src/network/HTTP.js

```diff
@@ -0,0 +1,40 @@
+import axios from 'axios';
+
+export default class HTTP {
+  constructor({ protocol = 'http', hostname = 'localhost', port = 8181, token }) {
```

Comment:
```suggestion
  constructor({ protocol = 'http', hostname = 'localhost', port = 80, token }) {
```

Reply:
Done! Also change the other params to the production cloud as default

Reply:
If default is now https, the port must be changed to 443.

Reply:
Done!